docs/elasticsearch/cluster.md

```diff
@@ -198,6 +189,135 @@ x.x.x.x           23          38   0    0.12    0.07     0.06 md        -      o
+
+This architecture can help you save a lot of money in storage costs because you can increase data retention by increasing the number of warm nodes and add hot nodes based on the total volume of data coming in.
+
+To configure a hot/warm storage architecture, add `temp` attributes to `odfe-d1` and `odfe-d2`, respectively:
+
+```yml
+node.attr.temp: hot
+```
+```yml
+node.attr.temp: warm
+```
+
+You can set the attribute name and value to whatever you want as long as it’s consistent for all your hot and warm nodes.
+
+To add an index `newindex` to the hot node:
+
+```json
+PUT newindex
+{
+  "settings": {
+    "index.routing.allocation.require.temp": "hot"
+  }
+}
+```
+
+If you take a look at the shard allocation for `newindex`:
+
+```json
+GET _cat/shards/newindex?v
+index     shard prirep state      docs store ip         node
+new_index 2     p      STARTED       0  230b 10.0.0.225 odfe-d1
+new_index 2     r      UNASSIGNED
+new_index 3     p      STARTED       0  230b 10.0.0.225 odfe-d1
+new_index 3     r      UNASSIGNED
+new_index 4     p      STARTED       0  230b 10.0.0.225 odfe-d1
+new_index 4     r      UNASSIGNED
+new_index 1     p      STARTED       0  230b 10.0.0.225 odfe-d1
+new_index 1     r      UNASSIGNED
+new_index 0     p      STARTED       0  230b 10.0.0.225 odfe-d1
+new_index 0     r      UNASSIGNED
+```
+
+All primary shards are allocated to `odfe-d1`, which is our hot node. All the replica shards are unassigned because we’re forcing this index to allocate only to hot nodes.
+
+To add an index `oldindex` to the warm node:
+
+```json
+PUT oldindex
+{
+  "settings": {
+    "index.routing.allocation.require.temp": "warm"
+  }
+}
+```
+
+The shard allocation for `oldindex`:
+
+```json
+GET _cat/shards/oldindex?v
+index     shard prirep state      docs store ip        node
+old_index 2     p      STARTED       0  230b 10.0.0.74 odfe-d2
+old_index 2     r      UNASSIGNED
+old_index 3     p      STARTED       0  230b 10.0.0.74 odfe-d2
+old_index 3     r      UNASSIGNED
+old_index 4     p      STARTED       0  230b 10.0.0.74 odfe-d2
+old_index 4     r      UNASSIGNED
+old_index 1     p      STARTED       0  230b 10.0.0.74 odfe-d2
+old_index 1     r      UNASSIGNED
+old_index 0     p      STARTED       0  230b 10.0.0.74 odfe-d2
+old_index 0     r      UNASSIGNED
+```
+
+You can see the same thing only that all the primary shards are allocated to `odfe-d2`. Again, all replica shards are unassigned because we only have one warm node.
+
+You can configure your [index templates](../index-templates/) to set the `index.routing.allocation.require.temp` value to `hot`. So that your most recent data is indexed to your hot nodes.
+
+You can then use the [Index State Management (ISM)](../../ism/index/) plugin to periodically check the age of an index and once it reaches a certain age update this setting to warm nodes. This will automatically reallocate your data from your hot nodes to your warm nodes.
```

Comment:
I think we can lose the last sentence here.

and, once it reaches a certain age, change this setting to automatically move your data from hot nodes to warm nodes.